app/selenium_ui/conftest.py

```diff
@@ -71,6 +71,7 @@ def __read_input_file(file_path):
 
 selenium_results_file = ENV_TAURUS_ARTIFACT_DIR / 'selenium.jtl'
 selenium_error_file = ENV_TAURUS_ARTIFACT_DIR / 'selenium.err'
+webdriver_visible = os.getenv('WEBDRIVER_VISIBLE', False)
```

Comment:
It feels like we do not need this env variable at all now, because we use .yml file for this configuration.
Let's remove it completely.

Reply:
Also, need to clean README files in docs folder from this env variable.